resources/definitions/xyzprinting_da_vinci_jr_1p0a_pro.def.json

```diff
@@ -1,80 +1,59 @@
 {
-    "version": 2,
+    "version": 2, 
     "name": "XYZprinting da Vinci Jr. 1.0A Pro",
     "inherits": "xyzprinting_base",
-    "metadata":
-    {
-        "visible": true,
-        "author": "XYZprinting Software",
+    "metadata": {
+		"author": "XYZprinting Software",
         "manufacturer": "XYZprinting",
+        "visible": true,
         "file_formats": "text/x-gcode",
-        "exclude_materials": [
-            "generic_hips",
-            "generic_petg",
-            "generic_bam",
-            "ultimaker_bam",
-            "generic_pva",
-            "ultimaker_pva",
-            "generic_tough_pla",
-            "ultimaker_tough_pla_black",
-            "ultimaker_tough_pla_green",
-            "ultimaker_tough_pla_red",
-            "ultimaker_tough_pla_white",
-            "generic_cffcpe",
-            "generic_cffpa",
-            "generic_gffcpe",
-            "generic_gffpa",
-            "structur3d_dap100silicone",
-            "ultimaker_petg_blue",
-            "ultimaker_petg_grey",
-            "ultimaker_petg_black",
-            "ultimaker_petg_green",
-            "ultimaker_petg_white",
-            "ultimaker_petg_orange",
-            "ultimaker_petg_silver",
-            "ultimaker_petg_yellow",
-            "ultimaker_petg_transparent",
-            "ultimaker_petg_red_translucent",
-            "ultimaker_petg_blue_translucent",
-            "ultimaker_petg_green_translucent",
-            "ultimaker_petg_yellow_fluorescent",
-            "ultimaker_petg_red"
-        ],
-        "has_machine_quality": true,
+		"has_machine_quality": true,
         "has_materials": true,
         "has_variants": true,
-        "machine_extruder_trains": { "0": "xyzprinting_da_vinci_jr_1p0a_pro_extruder_0" },
-        "preferred_quality_type": "normal",
-        "preferred_variant_name": "Copper 0.4mm Nozzle",
-        "quality_definition": "xyzprinting_da_vinci_jr_1p0a_pro",
+		"exclude_materials": ["generic_hips", "generic_petg", "generic_bam", "ultimaker_bam", "generic_pva", "ultimaker_pva", "generic_tough_pla", "ultimaker_tough_pla_black", "ultimaker_tough_pla_green", "ultimaker_tough_pla_red", "ultimaker_tough_pla_white", "generic_cffcpe", "generic_cffpa", "generic_gffcpe", "generic_gffpa", "structur3d_dap100silicone", "ultimaker_petg_blue", "ultimaker_petg_grey", "ultimaker_petg_black", "ultimaker_petg_green", "ultimaker_petg_white", "ultimaker_petg_orange", "ultimaker_petg_silver", "ultimaker_petg_yellow", "ultimaker_petg_transparent", "ultimaker_petg_red_translucent", "ultimaker_petg_blue_translucent", "ultimaker_petg_green_translucent", "ultimaker_petg_yellow_fluorescent", "ultimaker_petg_red" ],
```

Comment:
Hey, if you set `"material_diameter": { "default_value": 1.75 } ` Then you don't have to exclude all of these materials because they are all 2.85 diameter materials.